### ISSUE TITLE
Move End user Features or Functionality Overview to a separate category?

### ISSUE THREAD

**Issue by wwalc** [ckeditor/ckeditor-dev#2992]
The Guides section is indeed huge. Howe about extract from it end user features or the whole Functionality Overview to a separate place that would have its own link in the primary navigation?

If the Angular and React integrations stay in their current place (in the Integration section in Guides), it would move them more to the top thanks to that (related: https://github.com/ckeditor/ckeditor-dev/issues/2990)

**Comment by jacekbogdanski** [comment]
I'm feeling like if category labels would be folded by default, the whole navigation could be more visible.

**Comment by wwalc** [comment]
True, at the same time things like Ctrl + F "Angular" wouldn't work then.

**Comment by jacekbogdanski** [comment]
Maybe something like `fold/unfold all` button could do the magic?

**Comment by f1ames** [comment]
I think we may try with extracting whole **Functionality Overview** section and see how it works.

**Comment by f1ames** [comment]
See also @AnnaTomanek https://github.com/ckeditor/ckeditor-dev/issues/3084#issue-441067352 for some broader explanation.

**Comment by f1ames** [comment]
This issue basically requires changes in both `ckeditor-dev` (https://github.com/ckeditor/ckeditor-dev/pull/3109) and `ckeditor-docs` (https://github.com/ckeditor/ckeditor-docs/pull/249) repositories which means both needs to be released at the same time. Thus I'm moving this to `4.12` milestone.

**Comment by f1ames** [comment]
The changes were covered in #3109 and https://github.com/ckeditor/ckeditor-docs/pull/249. The PRs were already accepted and closed, but we still need redirects map as mentioned in https://github.com/ckeditor/ckeditor-docs/pull/249#pullrequestreview-250184818:

> It looks good, however we need one more thing - redirects from old links to new ones. Links in docs were updated, however this change will break every external link to the old section which is generally bad for users and sad for SEO. I guess it needs to be done somewhere on the server side so we need @vokiel assistance here. From our perspective we can prepare a map of links `old - new` so @vokiel could easily use it. @engineering-this could you take care of creating such file?

After redirects map is created (can be posted in this issue) we can close it.

---

Also changes introduced for this issue caused an issue with docs building (links validation errors) described in #3178.

**Comment by engineering-this** [comment]
Total links: 47.

There are 46 catalogues in `features/`
Plus `index.html`, so it sums to 47, should be OK, but could you check if it looks OK to you @f1ames?

```
	[ {
		from: 'guide/dev_features.html',
		to: 'features/index.html'
	}, {
		from: 'guide/dev_accessibility_checker.html',
		to: 'features/accessibility_checker.html'
	}, {
		from: 'guide/dev_autocomplete.html',
		to: 'features/autocomplete.html'
	}, {
		from: 'guide/dev_autogrow.html',
		to: 'features/autogrow.html'
	}, {
		from: 'guide/dev_balloontoolbar.html',
		to: 'features/balloontoolbar.html'
	}, {
		from: 'guide/dev_basicstyles.html',
		to: 'features/basicstyles.html'
	}, {
		from: 'guide/dev_bbcode.html',
		to: 'features/bbcode.html'
	}, {
		from: 'guide/dev_codesnippet.html',
		to: 'features/codesnippet.html'
	}, {
		from: 'guide/dev_codesnippetgeshi.html',
		to: 'features/codesnippetgeshi.html'
	}, {
		from: 'guide/dev_colorbutton.html',
		to: 'features/colorbutton.html'
	}, {
		from: 'guide/dev_copyformatting.html',
		to: 'features/copyformatting.html'
	}, {
		from: 'guide/dev_devtools.html',
		to: 'features/devtools.html'
	}, {
		from: 'guide/dev_drop_paste.html',
		to: 'features/drop_paste.html'
	}, {
		from: 'guide/dev_easyimage.html',
		to: 'features/easyimage.html'
	}, {
		from: 'guide/dev_media_embed.html',
		to: 'features/media_embed.html'
	}, {
		from: 'guide/dev_emoji.html',
		to: 'features/emoji.html'
	}, {
		from: 'guide/dev_enterkey.html',
		to: 'features/enterkey.html'
	}, {
		from: 'guide/dev_format.html',
		to: 'features/format.html'
	}, {
		from: 'guide/dev_fullpage.html',
		to: 'features/fullpage.html'
	}, {
		from: 'guide/dev_image.html',
		to: 'features/image.html'
	}, {
		from: 'guide/dev_image2.html',
		to: 'features/image2.html'
	}, {
		from: 'guide/dev_magicline.html',
		to: 'features/magicline.html'
	}, {
		from: 'guide/dev_mathjax.html',
		to: 'features/mathjax.html'
	}, {
		from: 'guide/dev_mentions.html',
		to: 'features/mentions.html'
	}, {
		from: 'guide/dev_notifications.html',
		to: 'features/notifications.html'
	}, {
		from: 'guide/dev_output_format.html',
		to: 'features/output_format.html'
	}, {
		from: 'guide/dev_pastefromexcel.html',
		to: 'features/pastefromexcel.html'
	}, {
		from: 'guide/dev_pastefromword.html',
		to: 'features/pastefromword.html'
	}, {
		from: 'guide/dev_placeholder.html',
		to: 'features/placeholder.html'
	}, {
		from: 'guide/dev_readonly.html',
		to: 'features/readonly.html'
	}, {
		from: 'guide/dev_removeformat.html',
		to: 'features/removeformat.html'
	}, {
		from: 'guide/dev_resize.html',
		to: 'features/resize.html'
	}, {
		from: 'guide/dev_sharedspace.html',
		to: 'features/sharedspace.html'
	}, {
		from: 'guide/dev_shortcuts.html',
		to: 'features/shortcuts.html'
	}, {
		from: 'guide/dev_size.html',
		to: 'features/size.html'
	}, {
		from: 'guide/dev_sourcearea.html',
		to: 'features/sourcearea.html'
	}, {
		from: 'guide/dev_spellcheck.html',
		to: 'features/spellcheck.html'
	}, {
		from: 'guide/dev_styles.html',
		to: 'features/styles.html'
	}, {
		from: 'guide/dev_tabindex.html',
		to: 'features/tabindex.html'
	}, {
		from: 'guide/dev_table.html',
		to: 'features/table.html'
	}, {
		from: 'guide/dev_toolbar.html',
		to: 'features/toolbar.html'
	}, {
		from: 'guide/dev_toolbarconcepts.html',
		to: 'features/toolbarconcepts.html'
	}, {
		from: 'guide/dev_toolbarlocation.html',
		to: 'features/toolbarlocation.html'
	}, {
		from: 'guide/dev_uicolor.html',
		to: 'features/uicolor.html'
	}, {
		from: 'guide/dev_uicolorpicker.html',
		to: 'features/uicolorpicker.html'
	}, {
		from: 'guide/dev_uilanguage.html',
		to: 'features/uilanguage.html'
	}, {
		from: 'guide/dev_uitypes.html',
		to: 'features/uitypes.html'
	} ];
```

**Comment by f1ames** [comment]
@engineering-this the list of redirects looks good.

The `tabindex.html` page is not linked from the menu (both for recent docs and after the change). I wonder if it's intended 🤔
**Update**: I see it's linked from keyboard shortcuts article so it's fine.

**Comment by f1ames** [comment]
Redirects added in https://github.com/ckeditor/ckeditor-docs/pull/252. Since it cannot be tested locally, will be checked after deploy. Closing this issue.
